app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :post
   belongs_to :user
+
+  def self.update_comment_counter(post_id)
+    post = Post.find_by(id: post_id)
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new comment has been saved to the database, then make the `update_comment_counter` method private